### PR TITLE
g++44 fixes

### PR DESCRIPTION
**Pull request by mleotta**
@mwoehlke-kitware I tried building on CentOS 5 with GCC 4.4 and had to work around some issues in the GUI code.  Let me know if you see any issues with these changes.

### CODE REVIEW

File: gui/CameraView.cxx [hunk 1]
```diff
@@ -580,11 +580,13 @@ void CameraView::setLandmarksData(kwiver::vital::landmark_map const& lm)
   }
 
   auto fields = QHash<QString, FieldInformation>{};
-  fields.insert("Elevation", {Elevation, {minZ, maxZ}});
```
Comment (mwoehlke-kitware) [inline gui/CameraView.cxx]: Does `insert("Elevation", {Elevation, FieldInformation{minZ, maxZ}});` work? I think that would be preferable to needing a named temporary.

Comment (mleotta) [inline gui/CameraView.cxx]: not as written, but `insert("Elevation", FieldInformation{Elevation, {minZ, maxZ}});` seems to work.  I'll revise tomorrow.

Comment (mleotta) [inline gui/CameraView.cxx]: done

**Comment by mwoehlke-kitware** [comment]
:+1: :shipit:

**Pull request merged**